integration-tests/README.md

```diff
@@ -24,14 +24,23 @@ Make sure the test apps have the latest local @embrace-io/react-native changes:
 npm run update-local-embrace
 ```
 
-Make sure the test apps are installed on the device/emulator before running tests:
+Make sure the test apps are installed on the device/emulator before running tests. Note that building the debug variant
+of the app may interfere with the tests as the debug menu gets in the way of UI elements.
+
+Android can run in release mode:
 
 ```bash
 cd basic-test-app
-npx expo run:android --variant debug # or release
-npx expo run:ios --variant debug # or release
+npx expo run:android --variant release
 ```
 
+For ios it doesn't apply the `--variant release` mode, so we need to do it through xcode:
```

Comment:
re https://github.com/embrace-io/embrace-react-native-sdk/pull/73/files#r1722342553 for setting it through the CLI

Reply:
updated